seawatch_registration/views/position.py

```diff
@@ -3,11 +3,14 @@
 from django.forms import CheckboxSelectMultiple
 from django.urls import reverse_lazy
 
-from seawatch_registration.mixins.model_form_widget_mixin import ModelFormWidgetMixin
+# TODO: put al
```

Comment:
Was soll diese Todo heißen

Reply:
Habe ich scheinbar nicht komplett gelöscht. Da stand vorher vermutlich, dass alle Mixins in eine Datei gepackt werden sollen. Das haben wir ja schon gemacht.

Reply:
Dann einfach weg damit :)